src/Sulu/Component/SmartContent/Orm/BaseDataProvider.php

```diff
@@ -151,21 +160,21 @@ function ($item) use ($locale) {
     }
 
     /**
-     * Decorates result as data item.
+     * Creates serialization context. Can be used to add own groups.
      *
-     * @param array $data
-     *
-     * @return array
+     * @return SerializationContext
      */
-    abstract protected function decorateDataItems(array $data);
+    protected function getSerializationContext()
+    {
+        return SerializationContext::create();
```

Comment:
in contact and account i can use full serialization of the object! but i think a method like this will be useable in other content types to add custom groups or other stuff.